tests/tests_suite_test.go

```diff
@@ -364,6 +366,18 @@ func waitUntilDeployed() {
 	deploymentTimedOut = false
 }
 
+func waitUntilTemplateValidatorIsRunning() {
```

Comment:
Can you move this function to `validator_test.go`, because it is only used there.